mimir-build-image/Dockerfile

```
@@ -8,7 +8,9 @@ FROM alpine/helm:3.8.2 as helm
 FROM golang:1.19.3-bullseye
 ARG goproxyValue
 ENV GOPROXY=${goproxyValue}
-RUN apt-get update && apt-get install -y curl python3-requests python3-yaml file jq zip unzip protobuf-compiler libprotobuf-dev shellcheck libpcap-dev skopeo && \
+RUN apt-get update && apt-get install -y curl python3-requests python3-yaml file jq zip unzip protobuf-compiler libprotobuf-dev shellcheck libpcap-dev
+# skopeo dependencies
+RUN apt-get install -y libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev pkg-config && \
```

Running these commands as separate `RUN` steps means that we'll end up with a small amount of extra gunk in the final image (`/var/lib/apt/lists/*`, `/tmp/*`, and `/var/tmp/*`), as these files will exist in the image layer for the first `RUN` and then be removed in the layer for the second `RUN`.

```suggestion
RUN apt-get update && apt-get install -y curl python3-requests python3-yaml file jq zip unzip protobuf-compiler libprotobuf-dev shellcheck libpcap-dev && \
    apt-get install -y libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev pkg-config && \ # skopeo dependencies
```

(syntax with the comment might not be quite right)

---

i'm sorry. github didn't show this comment when i was merging the PR

I tried your suggestion initially, but it didn't like the syntax as you say. I think the `\` is meant to escape the new line, but there is not new line. And putting a `/` at the end of the comment just treats it like a new line

---

If we remove the comment, newline should work. `\` needs to be at the end of line.

---

this should be a non-issue with https://github.com/grafana/mimir/pull/3973